src/OneDrive.Sdk.Authentication.WinStore/WebAuthenticationBrokerWebAuthenticationUi.cs

```diff
@@ -56,10 +56,10 @@ public class WebAuthenticationBrokerWebAuthenticationUi : IWebAuthenticationUi
             }
             else if (result != null && result.ResponseStatus == WebAuthenticationStatus.UserCancel)
             {
-                throw new ServiceException(new Error { Code = OAuthConstants.ErrorCodes.AuthenticationCancelled });
+                throw new ServiceException(new Error { Code = OAuthConstants.ErrorCodes.AuthenticationCancelled, Message = "Authentication cancelled by user."});
             }
 
-            throw new ServiceException(new Error { Code = OAuthConstants.ErrorCodes.AuthenticationCancelled });
+            throw new ServiceException(new Error { Code = OAuthConstants.ErrorCodes.AuthenticationCancelled, Message = "WebAuthenticationBroker.AuthenticateAsync() returned null." });
```

Comment:
Shouldn't there be a new error code to associate with this alternative message?  Is this a scenario that can happen- shouldn't we get an exception from WebAuthenticationBroker.AuthenicateAsync() and never null?